Source/Core/DolphinTool/ConvertCommand.cpp

```diff
@@ -248,28 +277,43 @@ int ConvertCommand::Main(const std::vector<std::string>& args)
   switch (format)
   {
   case DiscIO::BlobType::PLAIN:
+  {
     success = DiscIO::ConvertToPlain(blob_reader.get(), input_file_path, output_file_path,
                                      NOOP_STATUS_CALLBACK);
     break;
+  }
 
   case DiscIO::BlobType::GCZ:
-    success = DiscIO::ConvertToGCZ(blob_reader.get(), input_file_path, output_file_path,
-                                   volume->GetVolumeType() == DiscIO::Platform::WiiDisc ? 1 : 0,
+  {
+    u32 sub_type = std::numeric_limits<u32>::max();
```

Comment:
Is this an existing convention for non-GC/non-Wii GCZ files to have a sub_type of 0xFFFFFFFF, or did you make this up?

Reply:
I made it up. Dolphin has never had the ability to convert non-GC/Wii files to GCZ before, and there's no documentation on the format other than the source code.